### PR TITLE
#3311 Use OPENMODELICAHOME with generate-preassembled

### PR DESCRIPTION
**Pull request by barondim**
**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [ ] unit tests and non-regression tests were added (new model, new feature and bug fix)
- [ ] main documentation was updated (update of input/output file, 3rd party, model, repository organization, solver)
- [ ] example documentations were updated (new example in examples folder)
- [x] the corresponding milestone was added in the ticket and in this PR
- [ ] if this PR modifies the parameters or inputs/outputs of a model/solver: the corresponding xsl was added in util/xsl and platform DB were updated
- [ ] if this PR modifies a dictionary: the corresponding french dictionary was updated

### CODE REVIEW

File: util/envDynawo.sh [hunk 1]
```diff
@@ -1306,6 +1306,7 @@ generate_preassembled() {
   if ! is_launcher_installed; then
     install_launcher || error_exit "Error during launcher installation."
   fi
+  export OPENMODELICAHOME=$DYNAWO_INSTALL_OPENMODELICA
```
Comment (gautierbureau) [inline util/envDynawo.sh:1309]: export_var_env_force

**Comment by gautierbureau** [comment]
I think it would be better to put it in `set_environment`

File: util/envDynawo.sh (hunk 1 above)
Comment (barondim) [reply util/envDynawo.sh:1309]: If I use export_var_env_force, I get this error : "You must export variables with DYNAWO prefix for OPENMODELICAHOME."

**Review by gautierbureau — APPROVED** [review]

File: util/envDynawo.sh (hunk 1 above)
Comment (gautierbureau) [reply util/envDynawo.sh]: indeed

Comment (rosiereflo) [reply util/envDynawo.sh]: You could also add OPENMODELICA as an exception in the export_var_env_force

**Pull request merged**